agent/consul/usagemetrics/usagemetrics.go

```diff
@@ -137,4 +150,66 @@ func (u *UsageMetricsReporter) runOnce() {
 	}
 
 	u.emitServiceUsage(serviceUsage)
+
+	servers, clients := u.memberUsage()
+	u.emitMemberUsage(servers, clients)
+}
+
+func (u *UsageMetricsReporter) memberUsage() (int, map[string]int) {
+	if u.getMembersFunc == nil {
```

Comment:
I wonder if it would be better to add a nil check in the reporters constructor and error out there instead of reporting 0 metrics. I am having a hard time thinking of a case where it would be okay to create a reporter without one.

Reply:
I agree, will error out in the constructor